drbd_metrics_test.go

```diff
@@ -5,104 +5,111 @@ import (
 )
 
 func TestDrbdParsing(t *testing.T) {
-	var drbdDataRaw = []byte(` [
-		{
-		  "name": "1-single-0",
-		  "node-id": 2,
-		  "role": "Secondary",
-		  "suspended": false,
-		  "write-ordering": "flush",
-		  "devices": [
-			{
-			  "volume": 0,
-			  "minor": 2,
-			  "disk-state": "UpToDate",
-			  "client": false,
-			  "quorum": true,
-			  "size": 409600,
-			  "read": 0,
-			  "written": 548525,
-			  "al-writes": 4,
-			  "bm-writes": 0,
-			  "upper-pending": 0,
-			  "lower-pending": 0
-			} ],
-		  "connections": [
-			{
-			  "peer-node-id": 1,
-			  "name": "SLE15-sp1-gm-drbd1145296-node1",
-			  "connection-state": "Connected", 
-			  "congested": false,
-			  "peer-role": "Primary",
-			  "ap-in-flight": 0,
-			  "rs-in-flight": 0,
-			  "peer_devices": [
-				{
-				  "volume": 0,
-				  "replication-state": "Established",
-				  "peer-disk-state": "UpToDate",
-				  "peer-client": false,
-				  "resync-suspended": "no",
-				  "received": 548525,
-				  "sent": 0,
-				  "out-of-sync": 0,
-				  "pending": 0,
-				  "unacked": 0,
-				  "has-sync-details": false,
-				  "has-online-verify-details": false,
-				  "percent-in-sync": 100.00
-				} ]
-			} ]
-		}
-		,
-		{
-		  "name": "1-single-1",
-		  "node-id": 2,
-		  "role": "Secondary",
-		  "suspended": false,
-		  "write-ordering": "flush",
-		  "devices": [
-			{
-			  "volume": 0,
-			  "minor": 3,
-			  "disk-state": "UpToDate",
-			  "client": false,
-			  "quorum": true,
-			  "size": 10200,
-			  "read": 0,
-			  "written": 546005,
-			  "al-writes": 1,
-			  "bm-writes": 0,
-			  "upper-pending": 0,
-			  "lower-pending": 0
-			} ],
-		  "connections": [
-			{
-			  "peer-node-id": 1,
-			  "name": "SLE15-sp1-gm-drbd1145296-node1",
-			  "connection-state": "Connected", 
-			  "congested": false,
-			  "peer-role": "Primary",
-			  "ap-in-flight": 0,
-			  "rs-in-flight": 0,
-			  "peer_devices": [
-				{
-				  "volume": 0,
-				  "replication-state": "Established",
-				  "peer-disk-state": "UpToDate",
-				  "peer-client": false,
-				  "resync-suspended": "no",
-				  "received": 546005,
-				  "sent": 0,
-				  "out-of-sync": 0,
-				  "pending": 0,
-				  "unacked": 0,
-				  "has-sync-details": false,
-				  "has-online-verify-details": false,
-				  "percent-in-sync": 100.00
-				} ]
-			} ]
-		}]`)
+	var drbdDataRaw = []byte(`
+[
```

Comment:
why this doesn't go to fixtures?

Reply:
I didn't really change this particular test, I only indented the JSON, but I guess we could move this to a `.json` file and use the same for both this test and the one that invokes the fake executable. Do you want me to do that?

Reply:
yep. I think it is more consistent with other fixture metrics imho as approach. We can do also as follow-up or you can do it while I review :grin: